criu/cr-restore.c

```diff
@@ -3235,7 +3238,18 @@ rst_prep_creds_args(CredsEntry *ce, unsigned long *prev_pos)
 	memcpy(args->cap_prm, ce->cap_prm, sizeof(args->cap_prm));
 	memcpy(args->cap_bnd, ce->cap_bnd, sizeof(args->cap_bnd));
 
-	if (ce->n_groups) {
+	n = getgroups(0, NULL);
```

Comment:
@mclapinski should we extract this logic into a standalone function?

Reply:
Probably yes. This function is long enough.